exercises/forth/cases_test.go

```diff
@@ -0,0 +1,223 @@
+package forth
+
+// Source: exercism/x-common
+// Commit: 8804a76 Remove non-ASCII characters tests from forth exercise
+
+var testCases = []testCase{
+	// parsing and numbers
+	{
+		"empty input results in empty stack",
+		[]string{},
+		[]int{},
+	},
+	{
+		"numbers just get pushed onto the stack",
+		[]string{"1 2 3 4 5"},
+		[]int{1, 2, 3, 4, 5},
+	},
+	{
+		"all non-word characters are separators",
+		[]string{"1\x002\x133\n4\r5 6\t7"},
+		[]int{1, 2, 3, 4, 5, 6, 7},
+	},
+
+	// addition
+	{
+		"can add two numbers",
+		[]string{"1 2 +"},
+		[]int{3},
+	},
+	{
+		"errors if there is nothing on the stack",
```

Comment:
I see how to do it now. The template will generate separate test case slices, and I will use one main test function, and let it call a helper for the mechanics.